EpiAware/docs/src/showcase/replications/chatzilena-2019/index.jl

```diff
@@ -0,0 +1,622 @@
+
+As mentioned in _Chatzilena et al_ disease spread is frequently modelled in terms
+of ODE-based models. The study population is divided into compartments representing a specific stage of the epidemic status. In this case, susceptible, infected, and recovered individuals.
+
+```math
+\begin{aligned}
+{dS \over dt} &= - \beta \frac{I(t)}{N} S(t) \\
+{dI \over dt} &= \beta \frac{I(t)}{N} S(t) - \gamma I(t) \\
+{dR \over dt} &= \gamma I(t). \\
+\end{aligned}
+```
+where S(t) represents the number of susceptible, I(t) the number of
+infected and R(t) the number of recovered individuals at time t. The
+total population size is denoted by N (with N = S(t) + I(t) + R(t)), β
+denotes the transmission rate and γ denotes the recovery rate.
+
+"
+
+# ╔═╡ 0e78285c-d2e8-4c3c-848a-14dae6ead0a4
+md"
+We can interface to the `SciML` ecosystem by writing a function with the signature:
+
+> `(du, u, p, t) -> nothing`
+
+Where:
+- `du` is the _vector field_ of the ODE problem, e.g. ${dS \over dt}$, ${dI \over dt}$ etc. This is calculated _in-place_.
+- `u` is the _state_ of the ODE problem, e.g. $S$, $I$, etc.
+- `p` is an object that represents the parameters of the ODE problem, e.g. $\beta$, $\gamma$.
+- `t` is the time of the ODE problem.
+
+We do this for the SIR model described above in a function called `sir!`:
+"
+
+# ╔═╡ ab4269b1-e292-466f-8bfb-713d917c18f9
+function sir!(du, u, p, t)
+    S, I, R = u
+    β, γ = p
+    du[1] = -β * I * S
+    du[2] = β * I * S - γ * I
+    du[3] = γ * I
+
+    return nothing
+end
+
+# ╔═╡ f16eb00b-2d77-45df-b767-757fe2f5674c
+md"
+We combine vector field function `sir!` with a initial condition `u0` and the integration period `tspan` to make an `ODEProblem`. We do not define the parameters, these will be defined within an inference approach.
+"
+
+# ╔═╡ d64388f9-6edd-414d-a191-316f75b35b2c
+md"
+
+## Data for inference
+
+There was a brief, but intense, outbreak of Influenza within the (semi-) closed community of a boarding school reported to the British medical journal in 1978. The outbreak lasted from 22nd January to 4th February and it is reported that one infected child started the epidemic and then it spread rapidly. Of the 763 children at the boarding scholl, 512 became ill.
+
+We downloaded the data of this outbreak using the R package `outbreaks` which is maintained as part of the [R Epidemics Consortium(RECON)](http://www. repidemicsconsortium.org).
+
+"
+
+# ╔═╡ 7c9cbbc1-71ef-4d81-b93a-c2b3a8683d53
+data = "https://raw.githubusercontent.com/CDCgov/Rt-without-renewal/refs/heads/446-add-chatzilena-et-al-as-a-replication-example/EpiAware/docs/src/showcase/replications/chatzilena-2019/influenza_england_1978_school.csv2" |>
+       url -> CSV.read(download(url), DataFrame) |>
+              df -> @transform(df,
+    :ts=(:date .- minimum(:date)) .|> d -> d.value + 1.0,)
+
+# ╔═╡ aba3f1db-c290-409c-9b9e-6065935ede54
+N = 763;
+
+# ╔═╡ bb07a580-6d86-48b3-a79f-d2ed9306e87c
+sir_prob = ODEProblem(
```

Comment:
is it useful here to talk a bit about the ODEProblem object or link somewhere that does